src/apps/chifra/pkg/names/writer.go

```diff
@@ -0,0 +1,105 @@
+package names
```

Comment:
I'd like to think that this could somehow use StreamMany and FILE writer. Would that be possible? Otherwise, we have another type specific chunk of code to maintain if we add or remove fields.